build/charts/antrea/crds/antreaagentinfo.yaml

```diff
@@ -13,7 +13,99 @@ spec:
       schema:
         openAPIV3Schema:
           type: object
-          x-kubernetes-preserve-unknown-fields: true
+          properties:
+            version:
+              type: string
+            agentConditions:
+              type: array
+              items:
+                type: object
+                required:
+                - type
+                - status
+                - lastHeartbeatTime
+                properties:
+                  lastHeartbeatTime:
+                    type: string
+                    format: date-time
+                  type:
+                    type: string
+                    enum: ['AgentHealthy', 'ControllerConnectionUp', 'OVSDBConnectionUp', 'OpenflowConnectionUp']
+                  status:
+                    type: string
+                    enum: ['True', 'False', 'Unknown']
+                  reason:
+                    type: string
+                  message:
+                    type: string
+            localPodNum:
+              type: integer
+              format: int32
+            nodePortLocalPortRange:
+              type: string
+            nodeSubnets:
+              type: array
+              items:
+                type: string
+            apiCABundle:
+              type: string
```

Comment:
I believe this should also have a `format: byte` property, given that the Go type is `[]byte` and it will be base64-encoded when it is serialized.

see https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/

Reply:
updated